### DISCUSSION TITLE
Question: Mod support?

### DISCUSSION THREAD

**Discussion by perfectsine** [wolveix/satisfactory-server#8] [Q&A] [answered]
Hey before I go down this path, I'm wondering if mods are supported or have been tested?

**Comment by wolveix** [comment]
Hey! Sorry, I thought I replied to this. I haven't tested mods, but if they work in normal multiplayer, then there's no reason that they shouldn't work here.

**Comment by spyzzz** [comment]
You said there is no reason it should not work but you didn't implemented it right?

I can contribute, but i'm not sure yet how exactly add mod support to the server. Tried to directly put the SML like i di on my PC for local game. But no effect. Except that i need to have SML on ma PC to connect on my nas server now (So there must be someting installed i think, maybe SML is active, but the server is not able to load modules)

**Comment by spyzzz** [comment, marked as answer]
OK I was finally able to make mods running at least 2 that were multiplayer compatible (permaday & x3-sign)

It was just a bad folder destination on the server ;)

i has to be like this : 

for SML installation : 

/config/gamefiles/FactoryGame/Mods/SML/Sml.uplugin
/config/gamefiles/FactoryGame/Mods/SML/Binaries
/config/gamefiles/FactoryGame/Mods/SML/Content
/config/gamefiles/FactoryGame/Mods/SML/Resources

And then for each desired mod : 

/config/gamefiles/FactoryGame/Mods/\<ModeName\>/\<modename\>.uplugin

each mod has to get his own folder in Mods, that was my first mistake :)

**Reply by ysm1mh** [reply]
just curious.  how does one install  SML and add the mods to it?

is there a more indepth tutorial or explenation on how to get mods working?
i am lost on how to do this.
